components/style/properties/shorthand/border.mako.rs

```diff
@@ -246,7 +246,13 @@ pub fn parse_border(context: &ParserContext, input: &mut Parser)
                                 try!(input.expect_delim('/'));
                                 border_image_outset::parse(context, input)
                             }).ok();
-                            Ok((w, o))
+                            
+                            if ${w}.is_none() && ${o}.is_none() {
```

Comment:
Still looks wrong. If it wasn't clear in my last comment, `${}` is used in the Python mako template as Python expression evaluation, and w and o here are Rust variables.